snapper.spec.in

```diff
@@ -279,4 +279,17 @@ A PAM module for calling snapper during user login and logout.
 /usr/lib/pam_snapper/*.sh
 %doc %{_mandir}/*/pam_snapper.8*
 
+%package testsuite
```

Comment:
I guess this would be the package name. Would not it be better to add snapper to the name (e.g., snapper-integration-tests)?

Reply:
Without the `-n` option, `%package` takes a *suffix* for the package name, so this is declaring the `snapper-testuite` package. See http://ftp.rpm.org/max-rpm/s1-rpm-subpack-spec-file-changes.html